SB/SB/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using CsvHelper;
+using NLog;
+using NLog.Config;
+using NLog.Fluent;
+using NLog.Targets;
+
+namespace SB
+
+{
+    
+    internal class Program
+    {
+        private static readonly ILogger logger = LogManager.GetCurrentClassLogger();
+        
+        public static void Main(string[] args)
+        {
+            var config = new LoggingConfiguration();
+            var target = new FileTarget { FileName = @"C:\Work\Logs\SupportBank.log", Layout = @"${longdate} ${level} - ${logger}: ${message}" };
+            config.AddTarget("File Logger", target);
+            config.LoggingRules.Add(new LoggingRule("*", LogLevel.Debug, target));
+            LogManager.Configuration = config;
+            logger.Info("Starting program");   
+           
+            
+            var pathToCvs = "C:\\Work\\Bootcamp\\SB\\SB\\DodgyTransactions2015.csv";

```

Comment:
It's `pathToCsv` not `pathToCvs`, unless you meant [this](https://www.google.com/maps/dir//cvs/data=!4m6!4m5!1m1!4e2!1m2!1m1!1s0x40b1ff2e2eb53027:0x626b25cc2633edfe?sa=X&ved=2ahUKEwjootvrh9L4AhXResAKHafpANEQ9Rd6BAgFEAM)

Reply:
:))))))))))))))